napari/_qt/qt_viewer.py

```diff
@@ -443,34 +438,15 @@ def _create_canvas(self) -> None:
     def _diconnect_theme(self):
         self.viewer.events.theme.disconnect(self.canvas._on_theme_change)
 
-    def _add_visuals(self) -> None:
-        """Add visuals for axes, scale bar, and welcome text."""
+    def _add_overlay(self, overlay):
```

Comment:
```suggestion
    def _add_overlay(self, overlay: Overlay):
```

Probably don't want to commit from github since you'll need the import too.